dartagnan/src/main/java/com/dat3m/dartagnan/verification/model/ExecutionModel.java

```diff
@@ -497,6 +522,6 @@ private void extractCoherences() {
             }
 
             coherenceMap.put(addr, Collections.unmodifiableList(sortedWrites));
-        }
+        }*/
```

Comment:
Can we get rid of this?

Reply:
Good catch, I totally forgot about this change.
I'm actually not sure about that one. 
The uncommented version works for any encoding of `co`, whereas the commented version assumes clock variables (IDL-encoding).
Now, the clock-variable version is surely cheaper to extract, so we should probably use it for the IDL-encoding.
That being said, model extraction is rarely an issue (except for MathSAT5).
Maybe I will make the `useSATEncoding`-variable of `RelCo` accessible, and then the `ExecutionModel` can decide on which one to use.

Reply:
sounds good